### DISCUSSION TITLE
Компонент  SimpleCell свойство after

### DISCUSSION THREAD

**Discussion by KirKMS** [VKCOM/VKUI#6680] [Q&A] [answered]
Здравствуйте! Столкнулся с такой проблемой.
Есть список SimpleCell
![image](https://github.com/VKCOM/VKUI/assets/100795538/43cff4f6-638d-44f2-a8c1-945ffe50c911)
При onClick по пользователю, переходит на его профиль.
В поле after добавил компонент Button  для удаления пользователя из  списка.  Там тоже весит onClick и метод удаления.
Проблема в том, что при клике по кнопке, меня перекидывает в профиль, а обработчик события Button не срабатывает.
Мучаюсь уже какой день, и не могу понять. В чём причина такого поведения.
Если навести на кнопку
![image](https://github.com/VKCOM/VKUI/assets/100795538/13462694-4085-4229-8a5f-493162b1f2dc)
То hover компонента SimpleCell не срабатывает, что даёт понять, что кнопка должна отработать так  как нужно.

Подскажите пожалуйста

**Comment by SevereCloud** [comment, marked as answer]
Привет, нужно вызвать [preventDefault](https://developer.mozilla.org/ru/docs/Web/API/Event/preventDefault) для события нажатия кнопки

```jsx
<SimpleCell
  href="#"
  after={
    <Button onClick={(e)=>{e.preventDefault()}}>
      Удалить
    </Button>
  }
>
  Игорь Фёдоров
 </SimpleCell>
 ```

**Reply by KirKMS** [reply]
Привет, спасибо за помощь. Не знаю как я не додумался. Вроде банальные вещи
